### PR TITLE
[BUGFIX] Make Import and Truncate commands work with TYPO3 11

### PR DESCRIPTION
**Pull request by linawolf**

### CODE REVIEW

File: Classes/Command/ImportCommand.php [hunk 1]
```diff
@@ -51,6 +52,7 @@ protected function configure(): void
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
+        Bootstrap::initializeBackendAuthentication();
```
Comment (sabbelasichon) [inline Classes/Command/ImportCommand.php:55]: Is this really needed?

File: Classes/Domain/Manager/AbstractManager.php [hunk 2]
```diff
@@ -132,6 +132,7 @@ public function persist(): void
             $this->dataHandler->process_datamap();
             if (count($this->dataHandler->errorLog) !== 0) {
                 $this->logger->error('Error in importing', $this->dataHandler->errorLog);
```
Comment (sabbelasichon) [inline Classes/Domain/Manager/AbstractManager.php]: I think we can remove the logging part here. The exception is catched by the global ErrorHandler anyways.

File: Classes/Command/ImportCommand.php (hunk 1 above)
Comment (linawolf) [reply Classes/Command/ImportCommand.php:55]: In my Project context this did not work without initializing the user

File: Classes/Domain/Manager/AbstractManager.php (hunk 2 above)
Comment (linawolf) [reply Classes/Domain/Manager/AbstractManager.php]: done

**Pull request merged**